thoth/storages/graph/postgres.py

```diff
@@ -6710,6 +6710,45 @@ def get_bloat_data(self) -> List[Dict[str, Any]]:
 
         return self._process_bloat_data_results(tables=tables)
 
+    def store_old_solver_documents(self, os_name=None, os_version=None, python_version=None):
+        """Stores old solver documents to be deleted in Ceph"""
+        solver_store = SolverResultsStore()
+        solver_store.connect()
+
+        for document_id in solver_store.get_document_listing():
+            document_id_requirements = tuple(document_id.split("-")[1:4])
+
+            if document_id_requirements == (os_name, os_version, python_version):
+                json_document = solver_store.retrieve_document(document_id)
+                solver_store = store_document(json_document)
+                _LOGGER.info(f"Stored to be deleted document {document_id} in Ceph")
+
+    def store_old_adviser_documents(self, end_datetime=None, adviser_version=None):
+        """Stores old adviser documents to be deleted in Ceph"""
+        solver_store = SolverResultsStore()
```

Comment:
Copy-pasta - also for the package-extract docs below:

```suggestion
        adviser_store = AdviserResultsStore()
```

Reply:
Thanks, it is now fixed by : [Corrected value of adviser_store to AdviserResultsStore() at L6728](https://github.com/thoth-station/storages/pull/2468/commits/16a818dc4e8961111dcaa06f6d800767963935e0)